apps/coscrad-frontend/src/components/transcripts/transcript-presenter.tsx

```diff
@@ -18,7 +18,7 @@ export const TranscriptPresenter = ({ transcript }): JSX.Element => {
                 {isNullOrUndefined(transcript)
                     ? null
                     : transcript.participants.map((item: ITranscriptParticipant) => (
-                          <Box>
+                          <Box key={item.name.replace(' ', '-')}>
                               <Typography variant="body1" sx={{ fontWeight: 'bold' }}>
```

Comment:
I believe you don't need sx override here,
`
<Typography variant="body1" fontWeight={'bold'}>`

Reply:
Ok, this was a question I had - from what I could see the MUI theme Variants don't include a "bold" option.  All of `body1`, `body2` and `subtitle` are regular fontweight (see: [https://mui.com/system/typography/#variant](https://mui.com/system/typography/#variant)), and the rest of the variants are headings.  

Looking at the docs, I guess they want us to use Box for bold: [https://mui.com/system/typography/#font-weight](https://mui.com/system/typography/#font-weight) within the `Typography` tag (as long as the Typography tag is `component="div"`).

Reply:
Okay, I see what you mean in the the docs. Were you thinking this could be a use case where we'd use,

`<Box component={Typography} fontWeight={'bold'}>`

I was thinking it could cut down on the JSX/HTML.

Reply:
This is where I've ended up for now.  I realized that the null or undefined check for `transcript` needed to be moved up to the parent component and that simplified things a bit.  And then treating the `Typography` as blocks rather than inline I'm trying it out with the `Box` example from the docs, see what you think.

<img width="864" alt="image" src="https://github.com/COSCRAD/coscrad/assets/63014783/ca758f92-346b-4589-b050-87a1a9dcf94b">